doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_topology.rst

```diff
@@ -0,0 +1,478 @@
+
+How Manifest Class ID - based filtering works
+---------------------------------------------
+
+The SDFW has access to a list of Manifest Class ID UUIDs supported by the device.
+During the update candidate evaluation process, the SDFW compares the Manifest Class ID of each candidate manifest against this list.
+If the Manifest Class ID of a candidate manifest is not recognized by the device, the update candidate is rejected.
+
+Expressing Class-Specific-Information
+-------------------------------------
+
+Let's illustrate this with an example.
+``ACME Corp`` has two product types in portfolio: for example roller shutter motors and light bulbs.
+Both product types are powered by the nRF54H20 SoC and share the hardware design of the radio-related part, to the extent that they can share the same set of executable binary images for the radio domain.
+At the same time, hardware differences between motors and light bulbs are significant and it makes sense to have a separated set of executable binary images for the application domain.
+
+It is worth to consider assigning the same Manifest Class ID for Radio Local Manifest for both products, for example::
+
+   uuid5(acme_vid, 'nRF54H20_radio')
+
+and two distinct Manifest Class ID values for Application Local Manifest, for example::
+
+    uuid5(acme_vid, 'nRF54H20_light_bulb_app')
+
+    uuid5(acme_vid, 'nRF54H20_roller_shutter_app')
+
+.. figure:: ../../../../../../../modules/doc-internal/sw_arch/suit/images/suit_acme_manifests.png
+
+   Manifest hierarchies for ACME Corp devices.
+
+This approach allows you to share the same radio domain update package (executable binaries and SUIT manifest) across both products.
+Additionally, if an application domain update package intended for the light bulb is accidentally delivered to the roller shutter motor, the motor will reject it due to the unrecognized (unsupported) Manifest Class ID.
+
+Downgrade prevention policy
+===========================
+
+Product requirements can impose different downgrade prevention policies for different manifests.
+The following policy values are supported:
+
+* ``downgrade_prevention_disabled``
+* ``downgrade_prevention_enabled``
+
+How downgrade prevention works
+------------------------------
+
+The ``suit-manifest-sequence-number`` (an element of the SUIT manifest) is a monotonically increasing anti-rollback counter.
+As part of the update candidate evaluation process, the suit-manifest-sequence-number of the candidate manifests is compared with the installed counterpart.
+The result of this comparison, along with the associated downgrade prevention policy, determines whether a candidate is accepted or rejected.
+
+Assuming ``downgrade_prevention_enabled`` is enabled, a candidate manifest will be accepted only if its ``suit-manifest-sequence-number`` is not lower than the ``suit-manifest-sequence-number`` of the installed manifest.
```

Comment:
"now lower" implies that equal number is accepted. Is that intentional?

Reply:
@wiba-nordic Are you referring to "not lower"?

Reply:
Yes. I don't know what the intended functionality is, but having "not lower then" is the same as "the same or higher as". So if the candidate manifest is accepted when its `suit-manifest-sequence-number` is the same as the installed manifest's, this is correct.